blocks/library/text/index.js

```diff
@@ -47,7 +47,25 @@ registerBlock( 'core/text', {
 		}
 	],
 
-	edit( { attributes, setAttributes, insertBlockAfter, focus, setFocus } ) {
+	formatting: [
```

Comment:
In [API docs](https://github.com/WordPress/gutenberg/tree/master/blocks#api) I'd remarked that "Inline controls for Editable elements are identical for every block and cannot be modified." The thinking being that it'd be quite rare for an editable to not need these standard formatting controls, even to the point that we'd want to discourage it for the break in consistency. Do you have any feelings on this? Or could we at least have a standard set and make the intent an opt-out instead of opt-in?